src/Cmts.ml

```diff
@@ -270,20 +270,20 @@ let partition_after_prev_or_before_next t ~prev cmts ~next =
         let same_line_as_next l =
           next.loc_start.pos_lnum = l.loc_start.pos_lnum
         in
-        let colon_before =
-          let loc_end =
-            {prev.loc_end with pos_cnum= prev.loc_end.pos_cnum - 1}
-          in
+        let char_before ~in_ =
+          let pos_cnum = prev.loc_end.pos_cnum - 1 in
+          let loc_end = {prev.loc_end with pos_cnum} in
           let char_loc = {prev with loc_start= loc_end} in
           let str = Source.string_at t.source char_loc in
-          String.equal str ";"
+          List.mem in_ str ~equal:String.equal
```

Comment:
I'm not fan of the `List.mem` approach, I'd prefer pattern matching or multiple `String.equal`

Reply:
done